.travis.yml

```diff
@@ -47,4 +51,4 @@ script:
 
 notifications:
   slack:
-    secure: buhKHDpBl9aOddo6uC6wZH7qMMwaz/QF7i6VKxxslRPQWdXTVkeWDvacDRyVW6dhXHPMPr+HDeELsqNi3VoQuBWcXWmr+vNry2lGaXqRJCPQ/vW0Nn95RyZ9PogJ+XiO8OwRXujnWqUETynHh3VnOJWskHfcCanZDl4TfSadWQclEZ+r0viXuoBPwzSNkqu/PIHQl9lG47+x1Quh1yq9uQ87JiGIHvZ1I2FxibpSVEW35GBxWfwo0YFH1MZYz6Lpvc9gd7i5knygvjgkvCbwVEhPy9fC+TCo6nMBh7mA9YH0RETtUs9f3vwgZHBrtMaSCsLfWq5s/+Tj7fkcQ0IBQSx9uCeIKl1GSqgK5I3vLwH4icb3txxLVxLcMJNEJxpfcPUUWFb6CYBbRndkakMFEw5uyHwDbN6PBqWqAyMfqbfGOR38bQGQTcz7onw5nNK5gNHGJ3eMRLzrid+c+BTPbrcFrX4KX8rOLnzkyTWsbJrRVfFqqIPQW+XrDLlcOgg8L1xrBBlMlGYkwuOfsgVZck8Iexv6JRpSECABBCcu/AlYx3lwfxqNqKZU+OQ1/sOv+mri7SXajSumoaSiX/0wL7mhnShAu9F6ZqEMry+evkR30ePOKz1nSKzTiQk0nmbKH2WdbZbS/kCJXFCYZTCzUqKPvk15G46vDabTy8WQCLk=
+    secure: $SLACK_SECURE_TOKEN
```

Comment:
When I execute `travis lint` , the command results following error:

```
[x] [warn] on notifications.slack.rooms.secure: invalid value on secure string: {:secure=>$SLACK_SECURE_TOKEN}
```

Furthermore, notifications not working even if I set an environment variable at Travis project page.
Does it work for your environment?

Reply:
It works only when i use the format below:

```yml
notifications:
  slack:
    rooms:
      - automation-resources:NvHiFcFKbJAZuFzF1QJzaUtk
      - automation-resources:blVjL2z7mntvT3Xrpk0WtQVZ#slack-notifications
```

All the times I used the cryptographic version it didn't work.